<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notes</class>
 <widget class="QWidget" name="Notes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>421</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>417</width>
         <height>427</height>
        </rect>
       </property>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>421</width>
          <height>431</height>
         </rect>
        </property>
       </widget>
       <widget class="QScrollBar" name="verticalScrollBar">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>140</y>
          <width>16</width>
          <height>160</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="BulletButton">
      <property name="text">
       <string>Bullet Point</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ItalicButton">
      <property name="text">
       <string>Italicize</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="StrikethroughButton">
      <property name="text">
       <string>Strikethrough</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BoldButton">
      <property name="text">
       <string>Bold</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ExportButton">
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
